core/src/main/java/com/epam/esm/dao/GiftTagDao.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.esm.dao;
+
+
+import com.epam.esm.model.entity.GiftTagEntity;
+
+import java.util.List;
+
+public interface GiftTagDao {
+    void insertNewGiftTagEntity(GiftTagEntity giftTagEntity);
+    List<GiftTagEntity> findAllGiftTagEntity();
```

Comment:
could please rename method like
saveGiftTag
and findAllGiftTags
I avoid entity in name as you dao is called as GiftTagDao